### PR TITLE
Adding note and example command for the  `Runnin more worksapces` chapter

### PR DESCRIPTION
**Pull request by MichalMaler**
Signed-off-by: Michal Maléř <mmaler@redhat.com>

> Read our [Contribution guide](https://github.com/eclipse/che-docs/blob/master/CONTRIBUTING.adoc) before submitting a PR.

### PR Checklist

As the author of this Pull Request I made sure that:

- [ ] `vale` has been run successfully against the PR branch
- [ ] Link checker has been run successfully against the PR branch
- [ ] Documentation describes a scenario that is already covered by QE tests, otherwise an issue has been created and acknowledged by Che QE team
- [ ] Changed article references are updated where they are used (or a redirect has been set up on the docs side):
    - [ ] Dashboard [branding.json](https://github.com/eclipse/che-dashboard/blob/master/src/components/branding/branding.json)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)

### CODE REVIEW

File: modules/end-user-guide/partials/assembly_workspaces-overview.adoc [hunk 1]
```diff
@@ -18,7 +18,19 @@ Pods manage each component of a {prod-short} workspace. Therefore, everything ru
...
 The embedded browser-based IDE is the point of access for everything running in a {prod-short} workspace. This makes a {prod-short} workspace easily shareable.
 
-IMPORTANT: By default, it is possible to run only one workspace at a time. To change the default value, see: xref:installation-guide:advanced-configuration-options-for-the-che-server-component.adoc#users-workspace-limits[Users workspace limits].
+[IMPORTANT]
+====
+By default, it is possible to run only one workspace at a time. To increase the number of concurrent workspaces a user can run, patch the checluster:
+
+[subs="+quotes,+attributes"]
+----
+oc patch checluster {prod-checluster} -n __<user-project-namespace>__ --type=merge \
```
Comment (yhontyk) [inline modules/end-user-guide/partials/assembly_workspaces-overview.adoc]:
```suggestion
$ oc patch checluster {prod-checluster} -n __<user-project-namespace>__ --type=merge \ 
```

File: modules/installation-guide/examples/system-variables.adoc [hunk 2]
```diff
@@ -276,9 +276,20 @@
...
+ `+CHE_LIMITS_USER_WORKSPACES_RUN_COUNT+`,"`+1+`","The maximum number of running workspaces that a single user is allowed to have. If the user has reached this threshold and they try to start an additional workspace, they will be prompted with an error message. The user will need to stop a running workspace to activate another."
 ,=== 
 
+[IMPORTANT]
+====
+By default, it is possible to run only one workspace at a time. To increase the number of concurrent workspaces a user can run, patch the checluster:
+
+[subs="+quotes,+attributes"]
+----
+oc patch checluster {prod-checluster} -n __<user-project-namespace>__ --type=merge \
```
Comment (yhontyk) [inline modules/installation-guide/examples/system-variables.adoc]:
```suggestion
$ oc patch checluster {prod-checluster} -n __<user-project-namespace>__ --type=merge \ 
```

**Review by yhontyk — APPROVED** [review]
Just two missing command-line prompts. Otherwise, LGTM, approving.

**Comment by MichalMaler** [comment]
Great, Yana! Thank you for the review. Nice catches.

**Review by ScrewTSW — APPROVED** [review]
LGTM

**Pull request merged**